src/main/webapp/js/common/react/tests/Navbar.test.js

```diff
@@ -2,59 +2,86 @@ import React from 'react';
 import Navbar from '../components/Navbar.js';
 import '@testing-library/jest-dom/extend-expect';
 import {render, screen} from '@testing-library/react';
-import fetchMock from 'fetch-mock-jest';
-import {makeRelativeUrlAbsolute} from '../../../fetch_handler.js';
 
 const navbarUrls = [
   {
     href: '/',
     name: 'Home',
+    requiresAuth: false,
   },
   {
     href: '/projects.html',
     name: 'Projects',
+    requiresAuth: false,
+  },
+  {
+    href: '/dashboard.html',
+    name: 'Dashboard',
+    requiresAuth: true,
   },
 ];
```

Comment:
is there a way we can reuse the urls variable from display_navbar.js?

Reply:
Also, why don't we call `display_navbar.js` `navbar.js`? It seems like display may be redundant (or I just don't get how it is different than all of the other javascript files in this codebase that ultimately display html).

Reply:
As for why is isn't called `navbar.js`, I wanted to avoid ambiguity between a `navbar.js` file and the `Navbar.js` file, where `Navbar.js` is the file containing the `Navbar` React component. `display_navbar.js` also makes it clear that it is not a React component and instead accomplishes the task of displaying the Navbar. I personally prefer `display_navbar.js`, but I'd be happy to hear more about why you might think otherwise!

Reply:
And for reusing the URLs available in `display_navbar.js`, that's totally doable, and I see how that could reduce the number of experimental variables present if the tests ever fail. I'll update that.